src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -105,6 +106,16 @@ export const ARShowScreen = ({ navigation, route }) => {
         <Icon.NamedIcon name="camera" color={colors.darkText} size={normalize(30)} />
       </TouchableOpacity>
 
+      <TouchableOpacity
+        style={[styles.animationButton, styles.generalButtonStyle]}
+        onPress={() => setStartAnimation(!startAnimation)}
+      >
+        {startAnimation ? (
+          <Icon.NamedIcon name="stop" color={colors.primary} size={normalize(40)} />
+        ) : (
+          <Icon.NamedIcon name="play" color={colors.primary} size={normalize(40)} />
```

Comment:
why are these buttons bigger? should be 30 as well, right?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/578/commits/b6f83169a8c3647c64aa371c9815f19973fbfb81